in-room-controller/App.js

```diff
@@ -55,11 +55,16 @@ export default class App extends React.Component {
         const { remoteControlUrl } = this.state;
 
         if (remoteControlUrl) {
-            return (
-                <SideMenu menu = { this._renderSettingsMenu() }>
-                    <RemoteControl url = { remoteControlUrl } />
-                </SideMenu>
-            );
+            // The side menu for editing the URL is enabled only in the debug mode.
+            if (__DEV__) {
+                return (
+                    <SideMenu menu = { this._renderSettingsMenu() }>
+                        <RemoteControl url = { remoteControlUrl } />
+                    </SideMenu>
+                );
+            }
+
+            return <RemoteControl url = { remoteControlUrl } />;
```

Comment:
is there a better way of doing it ? is it possible to wrap into parent component conditionally ?

Reply:
okay I googled something, will update the PR

Reply:
process.env?